### PR TITLE
Adding Italian translation in config/locales

### PR DESCRIPTION
**Pull request by lpugin**
A new .yml file was added for Italian support

### CODE REVIEW

File: config/locales/blacklight.it.yml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+          one: 'Numero di scheda: %{title}'
+          other: 'Numeri di scheda'
+        url: 'URL: %{url}'
+        message: 'Messaggio: %{message}'
+
+      success: "Messaggio inviato"
+
+      errors:
+        to:
+          invalid: 'L'indirizzo e-mail inserito non è valido'
```
Comment (cbeer) [inline config/locales/blacklight.it.yml]: You should escape the single quote here (and line 123). Per the [YAML spec](http://yaml.org/spec/current.html#id2534365), I believe you should use `''` (two single quotes) here, or change the surrounding single quotes to double quotes.

**Comment by cbeer** [comment]
Fantastic, thank you. There's a couple escaping errors in the YAML, but once you fix those up, we can merge this.

**Comment by cbeer** [comment]
Looks good, thanks. Can I trouble you to rebase your commits down to a single commit? If not, I'll take care of it on our end.

**Comment by lpugin** [comment]
I'd be happy to do it but I am actually not sure what this means. Any hint?

**Comment by jcoyne** [comment]
@lpugin Take a look at https://help.github.com/articles/about-git-rebase/  @cbeer is asking you to `squash` your second commit into the first commit.

**Comment by cbeer** [comment]
Thanks @lpugin. I've merged this in c90ffb36a644a0395d5a4da9adacd9f42a4d82d2

**Pull request closed without merging**